.circleci/.circleci/.circleci/config.yml

```diff
@@ -0,0 +1,247 @@
+version: 2.1
```

Comment:
I think this belongs to `.circleci/.circleci/.circleci/.circleci/.circleci/.circleci`

Reply:
oh, fingers were fat during copy pasta :D